client/blocks/reader-full-post/index.jsx

```diff
@@ -396,6 +397,10 @@ const ConnectedFullPostView = connect(
 		if ( feedId ) {
 			props.feed = getFeed( state, feedId );
 		}
+
+		if ( ownProps.referral ) {
+			props.referralPost = getPostBySiteAndId( state, ownProps.referral.blogId, ownProps.referral.postId );
```

Comment:
we don't generally store posts in redux, only things that come in through related posts on the full post screen, and those get mirrored over into the flux `PostStore`. What case is this covering?